src/latex.jl

```diff
@@ -23,10 +23,16 @@ Can be added to `Luxor.jl` graphics via [`Video`](@ref).
 - **The `latex` method must be called from within an [`Object`](@ref).**
 
 # Arguments
+Positional:
 - `text::LaTeXString`: a LaTeX string to render.
 - `pos::Point`: position of the upper left corner of the latex text. Default: `O`
     - can be written as `x, y` instead of `Point(x, y)`
 - `object::Symbol`: graphics objects defined by `Luxor.jl`. Default `:stroke`.
+Keyword:
+- `valign::Symbol`: vertical alignment with respect to the specified `pos` parameter
```

Comment:
please mention the default value and move this below `Available objects:` as that describes the `object` parameter.